ci/build_image.sh

```diff
@@ -1,42 +1,30 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-source $(dirname $0)/env.sh
-
 print_help() {
     echo "Usage: ci/build_image.sh <build|push> <build|release>"
```

Comment:
I assume this needs an update as the second parameter is now unavailable

Reply:
Done :)